README.md

```diff
@@ -155,6 +155,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [WebFinery](https://webfinery.com/search) - Search the source code of the web
 - [Google Code Archive](https://code.google.com/archive/) - Data found on the Google Code Project Hosting Service, which was turned down in early 2016
 - [Snipplr](https://snipplr.com/all) - Code snippet search engine that allows users to search and share code snippets across various programming languages and frameworks
+- [Postman](https://www.postman.com/explore/collections) - Postman is an API platform for building and using APIs. Postman simplifies each step of the API lifecycle and streamlines collaboration so you can create better APIs—faster
```

Comment:
The description should be `Explore the best APIs, collections, workspaces in the world on the Postman Public API Network`

Reply:
@edoardottt  i have made the recommended changes.